features/step_definitions/visual_editor_steps.rb

```diff
@@ -53,9 +78,29 @@
   click_on "Save"
 end
 
-Then(/^I see the visual editor on subsequent edits of the HTML attachment$/) do
-  click_link "Edit attachment"
-  expect(page).to have_selector(".app-c-visual-editor__container")
-  expect(page).not_to have_selector(".app-c-govspeak-editor")
-  expect(page).to have_content("Any old iron")
+And(/^I exit the visual editor experience$/) do
+  click_on "Continue in markdown editor"
+end
+
+When(/^I edit a pre-existing publication$/) do
+  publication = create(:publication, visual_editor: nil)
+  visit edit_admin_publication_path(publication)
+end
+
+When(/^I update the publication in the govspeak editor$/) do
+  fill_in "edition_body", with: "Any old iron"
+end
+
+When(/^I update the publication in the visual editor$/) do
+  find(".ProseMirror").base.send_keys("Any old iron")
```

Comment:
This is interesting, bet it took some figuring out how to do

Reply:
Not when we have Demandadan.